_kbarticlespages/single-page-application.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: "Hosting Single Page Applications on Pages"
+date: November 2, 2023
+excerpt: 
+---
+
+## Intro
```

Comment:
```suggestion
```

Reply:
Removing this so the first lines render as a preview in the list: https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.sites.pages.cloud.gov/preview/cloud-gov/cg-site/KB/pages/knowledge-base/